.changeset/honest-impalas-hammer.md

```diff
@@ -0,0 +1,15 @@
+---
+'@sl-design-system/sanoma-learning': patch
+'@sl-design-system/itslearning': patch
+'@sl-design-system/bingel-int': patch
+'@sl-design-system/bingel-dc': patch
+'@sl-design-system/grid': patch
+'@sl-design-system/clickedu': patch
+'@sl-design-system/magister': patch
+'@sl-design-system/bingel': patch
+'@sl-design-system/kampus': patch
+'@sl-design-system/neon': patch
+'@sl-design-system/max': patch
+---
+
+checkbox and radio button tokens added
```

Comment:
Please start the line with an uppercase letter; this text get's added to the `CHANGELOG.md` file of the various packages